gslib/tests/test_wildcard_iterator.py

```diff
@@ -499,3 +501,14 @@ def testExistingDirNoFileMatch(self):
         self._test_wildcard_iterator(uri).IterAll(
             expand_top_level_buckets=True))
     self.assertEqual(0, len(res))
+
+  def testExcludeDir(self):
+    """Tests that the exclude regex will omit a nested directory."""
+    exp_uri_strs = self.root_files_uri_strs
+    uri = self._test_storage_uri(suri(self.test_dir, '*'))
+    et = (self.test_dir, re.compile('dir1'))
```

Comment:
Looks like this test was not updated?

Reply:
Still working on updating the test.

Reply:
OK the tests are now passing and everything is good.